libs/sr/localization/assets/locales/en/relic.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Should be able to add this to the datamine extracted hash data, instead of manually translating this. https://github.com/frzyc/genshin-optimizer/blob/master/libs/sr/dm-localization/src/executors/gen-locale/lib/hashData.ts

Reply:
To do this, open TextMapEN.json and ctrl-f the string you want, such as `"Planar Sphere"`. There will probably be multiple matches, just choose any and copy the key/hash into hashData.ts